hopper.sh

```diff
@@ -11,4 +11,5 @@ LIB=$(pwd)/lib
 export LD_LIBRARY_PATH=$LIB:$LD_LIBRARY_PATH
 export DYLD_LIBRARY_PATH=$LIB:$DYLD_LIBRARY_PATH
 
-java -Xmx8g -jar $JAR $@
+echo $DYLD_LIBRARY_PATH
```

Comment:
I don't believe so. I wouldn't merge this yet though since I don't know if
it breaks installation on Linux or older osx.
On May 18, 2016 6:31 AM, "Sam Blackshear" <notifications@github.com> wrote:

> In hopper.sh <https://github.com/cuplv/hopper/pull/4#discussion_r63693610>
> :
>
> > @@ -11,4 +11,5 @@ LIB=$(pwd)/lib
> >  export LD_LIBRARY_PATH=$LIB:$LD_LIBRARY_PATH
> >  export DYLD_LIBRARY_PATH=$LIB:$DYLD_LIBRARY_PATH
> >
> > -java -Xmx8g -jar $JAR $@
> > +echo $DYLD_LIBRARY_PATH
>
> Is this needed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cuplv/hopper/pull/4/files/97fa46c84f318315a1b757e7dd89d79729417909#r63693610>
>